### ISSUE TITLE
No Dock counter on M1 Pro (Mac OS 12.3)

### ISSUE THREAD

**Issue by desdougan** [Ranchero-Software/NetNewsWire#3504]
I've just installed NNW 6.0.3 on my new M1 Pro MacBook Pro running Monterey. I imported my feeds via OPML from the older laptop and am seeing them update OK, other than the dock icon does not show the unread count.

I don't see any preference for this, so am wondering if this is a bug?

**Comment by Wevah** [comment]
Double-check that badging has been allowed in System Preferences -> Notifications.

**Comment by desdougan** [comment]
Yes, that was the issue, which I realized this morning. Sorry for the 
noise…


Des

--

Des Dougan
Principal
Dougan Consulting Group Inc.

O: 604-628-5434
C: 604-866-2848
E: ***@***.***
W: https://www.DouganConsulting.com

Peace of Mind, One Computer at a Time.


Remote Support URL:

https://sc.douganconsulting.com:8040

**Comment by Wevah** [comment]
No worries; it happens!
